helios-core/src/test/kotlin/helios/instances/InstancesJavaTimeTest.kt

```diff
@@ -26,7 +25,9 @@ class InstancesJavaTimeTest : UnitSpec() {
 
     "ZonedDateTime should be encoded and decoded successfully"{
       val now = ZonedDateTime.now()
-      ZonedDateTimeDecoderInstance().decode(ZonedDateTimeEncoderInstance().run { now.encode() }) should beRight(now)
+      ZonedDateTimeDecoderInstance().decode(ZonedDateTimeEncoderInstance().run { now.encode() }) should beRight(
```

Comment:
We should probably split these off in a Java8 module for Android support in a separate ticket? Looks like a great beginner ticket for interested contributors.

Reply:
https://github.com/47deg/helios/issues/126